lua/noice/config/cmdline.lua

```diff
@@ -20,7 +20,7 @@ function M.setup()
       local hl_group_border = "CmdlinePopupBorder" .. kind_cc
       Highlights.add(hl_group_border, "NoiceCmdlinePopupBorder")
 
-      format = vim.tbl_deep_extend("force", {
+      format = vim.tbl_deep_extend("keep", {
```

Comment:
This is not a solution, since this way, you can no longer override the default cmdline settings

Reply:
Got it. Still working on it.